initialization.py

```diff
@@ -0,0 +1,77 @@
+import os
+import requests
```

Comment:
Looks clean but please add a requirements.txt so I don't try to install requests instead of python-requests

Reply:
A requirements.txt might be overkill for just one package. But I should definetly update documentation. I'll write it in the README. Also, isn't it just the ``requests`` package? Never heard of ``python-requests``.